controller/user.js

```diff
@@ -63,7 +66,7 @@ async function getById(req, res) {
       code: 0,
       msg: "",
       data: {
-        user: buildUserJson(user)
+        user: formatUser(user)
       },
     });
   }
```

Comment:
`getById` is no longer needed. Refer to API docs. Remove this function.

Reply:
`getById` is use for GET /api/v1/me